OMEdit/OMEditLIB/Simulation/SimulationOutputWidget.cpp

```diff
@@ -575,6 +575,73 @@ void SimulationOutputWidget::compileModel()
 }
 
 
+/*!
+ * \brief SimulationOutputWidget::runPostCompilation
+ * Runs the post compilation command after the compilation of the model.
+ */
+void SimulationOutputWidget::runPostCompilation()
+{
+  const QString postCompilationCommand = OptionsDialog::instance()->getSimulationPage()->getPostCompilationCommand();
+  if (postCompilationCommand.size())
+  {
+    mpCompilationProcess = new QProcess;
```

Comment:
Use a new process object instead of overwriting the existing compilation process. Otherwise we have a memory leak. Delete the object in the destructor.

Reply:
ok, done